API/Services/ImageServices/IImage.cs

```diff
@@ -64,15 +64,15 @@ public interface IImage : IDisposable
     /// <param name="filename">The name of the file to save the image to.</param>
     /// <param name="format">The format to save the image in.</param>
     /// <param name="quality">The quality of the saved image.</param>
```

Comment:
Forgot to remove the quality comments. ;)

Reply:
Doh. :O Done